.ci/change_uuid.jl

```diff
@@ -0,0 +1,5 @@
+using UUIDs
+
+project_filename = joinpath(dirname(@__DIR__), "Project.toml")
+
+write("Project.toml", replace(read("Project.toml", String), r"uuid = .*?\n" =>"uuid = \"$(uuid4())\"\n"))
```

Comment:
https://github.com/JuliaLang/Pkg.jl/pull/2379

Reply:
Ah good catch. Yes we should just use the same UUID every time.

Reply:
This will be fixed in #2595